### ISSUE TITLE
Contributing

### ISSUE THREAD

**Issue by ben-page** [gibbed/Gibbed.Borderlands3#8]
I'm not sure if this delayed/stalled for technical or personal reasons, but I'm interesting in contributing.

Anyway, I have a decade of C# experience. I'm proficient with protocol buffers, encryption, binary file formats. I've contributed to other save game editors and trainers. But I'm only moderately proficient in reverse engineering.

**Comment by Reiss-Cashmore** [comment]
I've offered my help numerous times over the past few months. I just get ignored, you can see if you look at the other issues on this repo what his stance is. 

He essentially uses GitHub as a means for him to the release the software and that's about it. This project really doesn't embody open source values at all. Which is a shame really, I've never known developers to turn down so many free and experienced hands for completing software that is so many months behind self-set deadlines

**Comment by ensiform** [comment]
@Reiss-Cashmore What deadline was set exactly? Perhaps he's become less interested because there's been a lot of complaining and demanding people, which takes it's toll. :shrug:

**Comment by Reiss-Cashmore** [comment]
> @Reiss-Cashmore What deadline was set exactly? Perhaps he's become less interested because there's been a lot of complaining and demanding people, which takes it's toll. :shrug:

Did you even read my comment? Self-set. On his Twitter page. Numerous times. Missed his own dates. It's clear you read my message in a tone that you have projected. All I did was state the facts of his own words and actions. I had no negative spin. I only said it was a shame to see such support go ignored.

But I would be happy for you to tell me more about numerous people offering free help to finish his project can be considered complaining. Oh and please do enlighten me on how I'm part of the problem, after supporting gibbed monetarily for is editors? Can you say the same? 

I've never complained about him missing his own deadlines I was pointing out that he continues to refuse support despite that being the case. I'm not here to try and telepathically understand his mental state. I can only react to his actions. After repeatedly completely ignoring me when all I've sent is supportive messages offering help with anything UI based (After he mentioned on twitter the UI needs work). After seeing his comments on the other issues about how he refuses any help from others. As well as use the facade of an open-source project when really this is not. Then it really does not inspire and positivity around this project.

**Comment by pickledestroyer** [comment]
yass reiss, get his ass! 
![9F159343-5366-4F6A-AA12-EA3A992B5582](https://user-images.githubusercontent.com/60218287/72977779-4fb40300-3e29-11ea-9e4b-e7934c921660.gif)

**Comment by IOnlyFetchBranches** [comment]
God I love Github ahaha

**Comment by ensiform** [comment]
@Reiss-Cashmore I never implied you specifically were the problem though. I was simply stating what it appeared was happening before. I also offered some UI libs a few months ago that might help but did not see any response. Being open source does not mean he has to push all changes to public especially if they are not in a state he is comfortable with still.

**Comment by aprizm** [comment]
hey why dont we just band together and start an open source version of it. yes we will have to start from scratch but at least we wont have to wait like peasants lol

**Comment by pickledestroyer** [comment]
> hey why dont we just band together and start an open source version of it. yes we will have to start from scratch but at least we wont have to wait like peasants lol

yes ! end rick’s monopolistic chokehold on Borderlands save editors ! programmers of the world, unite ! 
![0928E2E5-FD2A-492B-B276-B53298DC8514](https://user-images.githubusercontent.com/60218287/73352646-ac119980-42e5-11ea-9862-b0f3e8d1afb2.jpeg)

**Comment by Atulin** [comment]
11 months since last code update, it's dead and buried. Did anybody actually start making an alternative editor? 🤣

**Comment by ensiform** [comment]
@Atulin there are other editors out there some cli based idk about full fledged guis with item editing though. From my understanding it is all thanks to gibbed's work even if he does not support this repo.
